README.rst

```diff
@@ -122,6 +122,6 @@ To build the Docker image::
     $ cd anitya/
     $ docker build -t anitya .
 
-To run the container (replace the value of DB_URL with the URL to connect to your production database)::
+To run the container, execute the command below. Be sure to replace the value of DB_URL with the URL to connect to your production database. Also ensure to replace SECRET_KEY with a 24 character hex value that is the same on every deployment of Anitya (used for session management)::
```

Comment:
I don't think there is a limit on the size of SECRET_KEY.

Also, nitpicking but could you cut the lines at ~80 chars? thanks :)